modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -2039,6 +2046,8 @@ def unique(self, keep="first", ignore_index=True, subset=None):
                 preserve_columns=True,
             )
         else:
+            # return self.to_pandas().squeeze(axis=1).unique() works faster
+            # but returns pandas type instead of query compiler
```

Comment:
Do we want to perform this in a separate issue? I guess so.

Reply:
https://github.com/modin-project/modin/issues/7182